texts/MANIFEST

```diff
@@ -3,3 +3,4 @@
 1   icarus      dolphingarlic   ACCESSED FROM: http://www.angelfire.com/dragon/erzulie/Icarus_Allsorts.htm                                      AUTHOR: Roger McGough           STYLE: Poem               LICENSE: Public Domain    YEAR: 1816
 1   kubla_khan  dolphingarlic   ACCESSED FROM: https://www.poetryfoundation.org/poems/43991/kubla-khan                                          AUTHOR: Samuel Taylor Coleridge STYLE: Poem               LICENSE: Creative Commons YEAR: 1968
 1   reggae      dolphingarlic   ACCESSED FROM: https://docs.google.com/document/d/1Rz7DNd8vFPaU_baNtyZVCIxAGUsTcamFnAxHT92AXBU/edit?usp=sharing AUTHOR: Andi Qu                 STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
+1   evolution   nu11us          ACCESSED FROM: https://pastebin.com/PUbdMj8q                                                                    AUTHOR: N/A                     STYLE: Informative Essay  LICENSE: Creative Commons YEAR: 2018
```

Comment:
So you wrote this article?  I thought I saw somewhere that it came from Wikipedia?

Reply:
I didn't. That is a pastebin with just the text and a link to the page in the title. See the first resolved code review.

Reply:
The `MANIFEST` should link to the original source.